examples/b2c/checkout/order-confirmation-email-flow/email/unfiled$public/B2B2C_Order_Confirmation_Email.email

```diff
@@ -9,25 +9,26 @@
          <img style="display: block; border: none; max-width: 50%; width:50%; margin-left: auto; margin-right: auto;" src="https://akashmar25.stmpa.stm.force.com/s/sfsites/c/cms/delivery/media/MCPHJLRYCNWJFDNMUOUB27ZI7QZY" />
         </div>
          <div style="padding: 0px 20px 20px;">
-            <h1  style="font-size:18px; font-weight:bold; padding:0px; margin: 0px;">Thank you for your order, {{{OrderSummary.OCE_Account_First_Name__c}}}!</h1>
-            <p style="padding:10px 0px; margin: 0px;">Please find your order details below.</p>
-            <h2 style="font-size:14px; font-weight:bold; padding:0px; margin: 0px;">Confirmation Number: {{{OrderSummary.OrderNumber}}}</h2>
+            <h1  style="font-size:20px; font-weight:bold; padding:0px; margin: 0px;">Thank you for your order, {{{OrderSummary.OCE_Account_First_Name__c}}}!</h1>
+            <p style="font-size:16px;padding:10px 0px; margin: 0px;">Please find your order details below.</p>
+            <h2 style="font-size:16px; font-weight:bold; padding:0px; margin: 0px;">Confirmation Number: {{{OrderSummary.OrderNumber}}}</h2>
+            <a style="display: block;background-color:#000;color:#ffffff;font-size:13px;font-weight:400;line-height:20px;padding:6px;text-align:center;text-decoration: none;margin-top:10px;" href={{{OrderSummary.OCE_Order_Url__c}}} title="View Order Status">View Order Status</a>
          </div>
          <hr style="border: 1px solid #CDCDCD; margin:0px;"/>
          <div style="padding:20px 10px 0px;">
             <table width="100%">
                 <tr style="vertical-align:top;">
                     <td width="50%">
                         <div style="padding: 0px 10px;">
-                        <h2 style="font-size:14px; font-weight:bold; padding:0px; margin: 0px;">Shipping Address</h2>
-                        <p style="padding:10px 0px; margin: 0px;">{{{OrderSummary.OCE_Shipping_Address__c}}}</p>
+                        <h2 style="font-size:16px; font-weight:bold; padding:0px; margin: 0px;">Shipping Address</h2>
+                        <p style="font-size:16px; padding:10px 0px; margin: 0px;">{{{OrderSummary.OCE_Shipping_Address__c}}}</p>
```

Comment:
Header & context with same font size?  Is it same in design?

Reply:
YEs, just the font weight difference only.